docs/example-scenario/apps/hpc-saas.md

```diff
@@ -0,0 +1,75 @@
+
+The company wants to expand the market for its applications by building a SaaS platform backed by cloud-based HPC technologies. Their customers should be able to pay for compute resources as needed and access massive computing power that would be unaffordable otherwise. The company's goals include:
+* Taking advantage of HPC capabilities in Azure to accelerate the product design and testing process
+* Using the latest hardware innovations to run complex simulations, while minimizing the costs for simpler simulations  
+* Enabling true-to-life visualization and rendering in a web browser, without requiring a high-end engineering workstation
+
+## Potential use cases
+
+Other scenarios using this architecture might include:
+
+* Genomics research
+* Weather simulation
+* Computational chemistry applications
+
+## Architecture
+
+![Architecture for a SaaS solution enabling HPC capabilities][architecture]
+
+* Users can access NV-series virtual machines (VMs) via a browser with a HTML5-based RDP connection using the Apache Guacamole service (http://guacamole.apache.org/). These VM instances provide powerful GPUs for rendering and collaborative tasks. Users can edit their designs and view their results without needing access to high-end mobile computing devices or laptops. The Altair PBSWorks or open-source PBS Professional scheduler spins up additional nodes based on user-defined heuristics and could alternatively leverage a solution based on Azure functions.
+* From a desktop CAD session, users can submit workloads for execution on available HPC cluster nodes. These workloads perform tasks such as stress analysis or computational fluid dynamics calculations, eliminating the need for dedicated  on-premises compute clusters. These cluster nodes can be configured to auto-scale based on load or queue depth based on active user demand for compute resources.
+* Azure Kubernetes Service (AKS) is used to host the web resources available to end users.
+
+### Components
+
+* [H-series virtual machines](/azure/virtual-machines/linux/sizes-hpc) are used to run compute-intensive simulations such as molecular modeling and computational fluid dynamics. The solution also takes advantage of technologies like remote direct memory access (RDMA) connectivity and InfiniBand networking.
+* [NV-series virtual machines](/azure/virtual-machines/windows/sizes-gpu) give engineers high-end workstation functionality from a standard web browser. These virtual machines have NVIDIA Tesla M60 GPUs that support advanced rendering and can run single precision workloads.
+* [General purpose virtual machines](/azure/virtual-machines/linux/sizes-general) running CentOS handle more traditional workloads such as web applications.
+* [Application Gateway](/azure/application-gateway/) load balances the requests coming into the web servers.
```

Comment:
Why App GW for load balancing vs Load Balancer? Is this for L7 routing or because AppGW has a WAF?

Also, can App GW connect directly to AKS? I thought you need an Azure LB to expose a k8s service.

Reply:
I'll follow up with the CSA re: the specific customer details on this.